addons/im_livechat/views/res_partner_views.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+        <record id="partner_livechat_sessions_action" model="ir.actions.act_window">
```

Comment:
I feel like it would be easier/cleaner to use an `object` action and to reuse the existing action. All we need is to enrich the domain with the agent we're opening the view for.